source/_components/sensor.transmission.markdown

```diff
@@ -2,90 +2,16 @@
 layout: page
 title: "Transmission Sensor"
 description: "Instructions on how to integrate Transmission sensors within Home Assistant."
-date: 2015-04-25 9:06
+date: 2018-12-12 22:59
 sidebar: true
 comments: false
 sharing: true
 footer: true
 logo: transmission.png
 ha_category: Downloading
-ha_release: pre 0.7
+ha_release: 0.85
```

Comment:
Seems incorrect, this sensor is still introduced `pre 0.7`, please revert this change.